src/vaadin-dialog.html

```diff
@@ -289,6 +391,172 @@
             e.preventDefault();
           }
         }
+
```

Comment:
These additions almost double the lines of code in this file (up to ~600). For clarity's sake, it might make sense to split these into dedicated mixin(s).

Reply:
Done.